.circleci/config.yml

```diff
@@ -67,10 +57,57 @@ workflows:
           requires:
             - build
 
-      - deploy:
-          name: deploy
+      # build and push chart to app catalog
+      - architect/push-to-app-catalog:
+          name: push-app-operator-to-control-plane-app-catalog
+          app_catalog: "control-plane-catalog"
+          app_catalog_test: "control-plane-test-catalog"
+          chart: "app-operator"
+          # Make sure docker image is successfully built.
+          requires:
+            - push-app-operator-to-quay
```

Comment:
Please check against the feedback in the chart-operator PR. We should wait for Aliyun as well.

https://github.com/giantswarm/chart-operator/pull/421/files#r409757080